### DISCUSSION TITLE
Button Component Not Showing App router

### DISCUSSION THREAD

**Discussion by BenParr64** [chakra-ui/chakra-ui#7835] [Q&A]
### Description

The buttons will flash on first load but after that they only show when hovered

Github Repo
https://github.com/BenParr64/chakra-ui-button-bug-demo

Deployed
https://chakra-ui-button-bug-demo.vercel.app/

### Link to Reproduction

https://chakra-ui-button-bug-demo.vercel.app/

### Steps to reproduce

1. Go to https://chakra-ui-button-bug-demo.vercel.app/)](https://chakra-ui-button-bug-demo.vercel.app/
2. Hover top left
3. See button error

### Chakra UI Version

 "@chakra-ui/next-js": "^2.1.4",     "@chakra-ui/react": "^2.7.1",

### Operating System

- [ ] macOS
- [ ] Windows
- [ ] Linux

**Comment by michachan** [comment]
> There may be problems in many components when using Chakra UI with the body tag in Layout.

Root layouts require html and body tags. Removing body tag will cause issues.

https://nextjs.org/docs/app/building-your-application/routing/pages-and-layouts#root-layout-required

**Comment by kelvin-du** [comment]
Please remove `globals.css`.
The problem is chakra styles get overridden by `tailwind`.

**Comment by onurhan1337** [comment]
@michachan That is a good point, thank you for sharing. For this problem @kelvin-du said a good solution but that doesn't feel enough yet. It is important that we can comfortably share like in Next12.

**Comment by BenParr64** [comment]
This still feels like a bug even with these solutions but thanks so much for the help everyone. Cheers!

Sent from Samsung Mobile on O2
Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by XxBrezxX** [comment]
Is this issue still active? I would like to contribute, can you give me more details?

**Comment by michachan** [comment]
> Is this issue still active? I would like to contribute, can you give me more details?

There may be a collision between the tailwind css declarations and the chakra provider setup for theme. If you'd like, you can start by investigating there. From what I can see, there's an issue with the base theme applying (loads at first then gets overridden- see the demo that OP posted).

Good luck!

**Comment by XxBrezxX** [comment]
I got an approach that fixes the bug visually but can be improved

**Comment by devjs1000** [comment]
@BenParr64 it is not an issue. it is because of you are using tailwind which reseting the css. 
read this https://tailwindcss.com/docs/preflight
just add this in your tailwind.config.js
```
corePlugins: {
  preflight: false,
}
```
